src/factors/BearingRange2D.jl

```diff
@@ -31,6 +21,16 @@ function getSample(cfo::CalcFactor{<:Pose2Point2BearingRange}, N::Int=1)
   return (smpls,)
 end
 
+
+function IIF.getParametricMeasurement(s::Pose2Point2BearingRange{Normal, Normal})
+
+  meas = [mean(s.bearing), mean(s.range)]
+  iΣ = [1/var(s.bearing)             0;
+                      0  1/var(s.range)]
+
+  return meas, iΣ
+end
```

Comment:
jip, this looks like the right way to go

Reply:
Hi, I pushed my local bugfix changes where I use `<:Normal`